src/lib.rs

```diff
@@ -22,7 +22,8 @@ pub struct LineEntry {
 }
 
 pub fn run(matches: clap::ArgMatches) -> Result<(), Error> {
-    let paths = dotenv_files(matches)?;
+    let mut paths = dotenv_files(matches)?;
+    paths.dedup();
```

Comment:
It doesn't work for the case:
```bash
$ dotenv-linter -i .my-env-file -i .env
.env:1 Leading space detected
.my-env-file:1 The line has spaces around equal sign
.env:1 Leading space detected
```

Reply:
I think you should use [HashSet](https://doc.rust-lang.org/std/collections/struct.HashSet.html) to fix that.

Reply:
Ah yes, because `dedup` only removes duplicates which follow after each other. Will fix :)

Reply:
A sort before the `dedup` would also work. But I think the HashSet is better.

Reply:
@mstruebing `sort` and `dedup` is better 😃
https://github.com/mgrachev/dotenv-linter/pull/68